patches/nginx-1.17.8-set_uri_security_issue.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/src/http/modules/ngx_http_static_module.c b/src/http/modules/ngx_http_static_module.c
+index 282d6ee..cb5b6c1 100644
+--- a/src/http/modules/ngx_http_static_module.c
++++ b/src/http/modules/ngx_http_static_module.c
```

Comment:
Do we have tests somewhere to cover this fix? Maybe in ngx_http_lua_module's test suite?